configure

```diff
@@ -129,22 +122,22 @@ CUDA_TOOLKIT_PATH="$CUDA_TOOLKIT_PATH"
 # directories separately.
 CUDNN_INSTALL_PATH="$CUDNN_INSTALL_PATH"
 
-# The Cuda SDK version that should be used in this build
+# The Cuda SDK version that should be used in this build (empty to use libcudart.so symlink)
 TF_CUDA_VERSION=$TF_CUDA_VERSION
 
-# The Cudnn version that should be used in this build
+# The Cudnn version that should be used in this build (empty to use libcudnn.so symlink)
 TF_CUDNN_VERSION=$TF_CUDNN_VERSION
 
 EOF
 
 function UnofficialSetting() {
   # Configure the Cuda toolkit version to work with.
-  perl -pi -e "s,CUDA_VERSION = '[0-9\.]*',CUDA_VERSION = '$TF_CUDA_VERSION',s" tensorflow/core/platform/default/build_config.bzl
-  perl -pi -e "s,(GetCudaVersion.*return )\"[0-9\.]*\",\1\"$TF_CUDA_VERSION\",s" tensorflow/stream_executor/dso_loader.cc
+  perl -pi -e "s,CUDA_VERSION = '[0-9\.]*',CUDA_VERSION = '.$TF_CUDA_VERSION',s" tensorflow/core/platform/default/build_config.bzl
```

Comment:
These two lines might go in a separate function, since it's not 'unofficial'

Also, line 176 probably should be removed....